online-unsafe-recovery.md

```diff
@@ -0,0 +1,84 @@
+> - It is recommended to perform the feature-related operations with the support of the TiDB team. If some incorrect operations are performed, it might be hard to recover the cluster.
+
+When permanently damaged replicas cause part of data to be unreadable and unwritable, you can use the online unsafe recovery feature to perform a lossy recovery operation.
+
+## Feature description
+
+In TiDB, data might be replicated in many stores at the same time according to the Placements Rules defined by users. This guarantees that data is still readable and writable even if a single or a few stores are temporarily offline or damaged. However, when most or all replicas of a Region are offline at the same time, the Region becomes temporarily unavailable, by design, to ensure data integrity.
+
+Suppose that multiple replicas of data encounter problems like permanent damage (such as disk damage), and this issue causes stores to fail to go online. In this case, this data is temporarily unavailable. Provided that you can accept data rewind or data loss, if you want the cluster back in use under this circumstance, TiDB can theoretically re-form the majority of replicas by manually overwriting the meta information of data shards. This allows application layer services to read and write (might be stale or empty) this data.
+
+In this case, if some stores with loss-tolerating data are permanently damaged, you can easily perform a lossy recovery operation by using online unsafe recovery. Using this feature, PD, under its global perspective, collects the meta information of data shards in all stores and generates a more real-time and more complete recovery plan. Then, PD distributes the plan to each surviving store to make the stores perform data recovery tasks. Also, after the data recovery plan is distributed, PD periodically checks the recovery progress to ensure that the current state of the cluster matches its expected state.
+
+## User scenarios
+
+Online unsafe recovery feature is suitable for the following scenarios:
+
+* The data for application services is unreadable and unwritable, because permanently damaged stores cause the stores to fail to restart.
+* You can accept data loss and want the affected data to be readable and writable.
+* You want to perform a one-stop online data recovery operation.
+
+## Usage
+
+### Prerequisites
+
+Before using online unsafe recovery, make sure the following things:
+
+* Part of data is indeed unavailable.
+* The offline stores cannot be automatically recovered or restarted.
+
+### Step 1. disable all types of scheduling
+
+You need to disable all types of internal scheduling, such as load balancing. After disabling them, it is recommended to wait for about 10 minutes so that the triggered scheduling can have sufficient time to complete the scheduled tasks.
```

Comment:
```suggestion
You need to temporarily disable all types of internal scheduling, such as load balancing. After disabling them, it is recommended to wait for about 10 minutes so that the triggered scheduling can have sufficient time to complete the scheduled tasks.
```